src/authentication/FlowDetails.tsx

```diff
@@ -0,0 +1,240 @@
+  const [liveText, setLiveText] = useState("");
+
+  useFetch(
+    async () => {
+      const flows = await adminClient.authenticationManagement.getFlows();
+      const flow = flows.find((f) => f.id === id);
+      const executions =
+        await adminClient.authenticationManagement.getExecutions({
+          flow: flow?.alias!,
+        });
+      return { flow, executions };
+    },
+    ({ flow, executions }) => {
+      setFlow(flow);
+      setExecutionList(new ExecutionList(executions));
+    },
+    [key]
+  );
+
+  const executeChange = async (
+    ex: AuthenticationFlowRepresentation,
+    change: LevelChange | IndexChange
+  ) => {
+    try {
+      let id = ex.id!;
+      if ("parent" in change) {
+        await adminClient.authenticationManagement.delExecution({ id });
+        const result =
+          await adminClient.authenticationManagement.addExecutionToFlow({
+            flow: change.parent?.displayName! || flow?.alias!,
+            provider: ex.providerId!,
+          });
+        id = result.id!;
+      }
+      const c = change as IndexChange;
+      const times = c.newIndex - c.oldIndex;
+      for (let index = 0; index < Math.abs(times); index++) {
+        if (times > 0) {
+          await adminClient.authenticationManagement.lowerPriorityExecution({
+            id,
+          });
+        } else {
+          await adminClient.authenticationManagement.raisePriorityExecution({
+            id,
+          });
+        }
+      }
+      refresh();
+      addAlert(t("updateFlowSuccess"), AlertVariant.success);
+    } catch (error) {
+      addAlert(
+        t("updateFlowError", {
+          error: error.response?.data?.errorMessage || error,
+        }),
+        AlertVariant.danger
+      );
+    }
+  };
+
+  const update = async (
+    execution: AuthenticationExecutionInfoRepresentation
+  ) => {
+    try {
+      await adminClient.authenticationManagement.updateExecution(
+        { flow: flow?.alias! },
+        execution
+      );
+      refresh();
+      addAlert(t("updateFlowSuccess"), AlertVariant.success);
+    } catch (error) {
+      addAlert(
+        t("updateFlowError", {
+          error: error.response?.data?.errorMessage || error,
+        }),
+        AlertVariant.danger
+      );
+    }
+  };
+
+  return (
+    <>
+      <ViewHeader
+        titleKey={toUpperCase(flow?.alias! || "")}
+        badges={[
+          { text: <Label>{t(usedBy)}</Label> },
+          builtIn
+            ? {
+                text: (
+                  <Label
+                    className="keycloak_authentication-section__usedby_label"
+                    icon={<CheckCircleIcon />}
+                  >
+                    {t("buildIn")}
+                  </Label>
+                ),
+                id: "builtIn",
+              }
+            : {},
+        ]}
+      />
+      <PageSection variant="light">
+        {executionList?.expandableList &&
+          executionList.expandableList.length > 0 && (
```

Comment:
To reduce the size of this statement you could use `length` as an implicit truthy or falsy value by letting JavaScript cast it for you:
```suggestion
        {executionList?.expandableList?.length && (
```